terraform/environments/example/secrets.tf

```diff
@@ -1,15 +1,41 @@
 ######################### Run Terraform via CICD ##################################
 # Get secret by name for environment management
+resource "random_password" "random_password" {
```

Comment:
Could you add this resource and the other secrets added here to a new `secrets.tf` file.  We should probably rename this file `platform_secrets.tf` as it contains data look ups for the main platform secrets. We'll need to amend the template to do the same as it is a bit confusing as to what to put here.